scripts/dev-e2e.sh

```diff
@@ -44,6 +35,15 @@ err() {
   echo -e "${RED}$1${NC}" >&2
 }
 
+log_group() {
+  if [ "${GITHUB_ACTIONS:-false}" == "true" ]; then echo -n "::group::"; fi
+  echo "$@"
+  while read -r line
+  do
+    echo "$line"
+  done
```

Comment:
I think this can be simplified as
```suggestion
  cat -
```